nx-X11/programs/Xserver/mi/Imakefile

```diff
@@ -179,6 +183,9 @@ NormalLibraryTarget(cbrt,cbrt.o)
 
 SpecialCObjectRule(miinitext,$(ICONFIGFILES),$(EXT_DEFINES))
 SpecialCObjectRule(miscrinit,$(ICONFIGFILES),$(EXT_DEFINES))
+SpecialCObjectRule(miglblt,$(ICONFIGFILES),$(FONT_DEFINES))
+SpecialCObjectRule(mipolytext,$(ICONFIGFILES),$(FONT_DEFINES))
```

Comment:
With mipolytext removed from Imakefile's SpecialCObjectRule(...) (i.e. not defining FONT_DEFINES, I get the build build failure:

```
mi/libmi.a(mipolytext.o): In function `miPolyText':
./nx-X11/programs/Xserver/mi/mipolytext.c:82: undefined reference to `dixGetGlyphs'
mi/libmi.a(mipolytext.o): In function `miPolyText8':
./nx-X11/programs/Xserver/mi/mipolytext.c:105: undefined reference to `dixGetGlyphs'
mi/libmi.a(mipolytext.o): In function `miPolyText16':
./nx-X11/programs/Xserver/mi/mipolytext.c:128: undefined reference to `dixGetGlyphs'
mi/libmi.a(mipolytext.o): In function `miImageText':
./nx-X11/programs/Xserver/mi/mipolytext.c:154: undefined reference to `dixGetGlyphs'
mi/libmi.a(mipolytext.o): In function `miImageText8':
./nx-X11/programs/Xserver/mi/mipolytext.c:176: undefined reference to `dixGetGlyphs'
mi/libmi.a(mipolytext.o):./nx-X11/programs/Xserver/mi/mipolytext.c:195: more undefined references to `dixGetGlyphs' follow
collect2: error: ld returned 1 exit status
```

So, it has to stay.